### ISSUE TITLE
[Bug Bash] The ‘Fix with GitHub Copilot’ link and ‘X’ button in the vulnerability InforBar at the top of PM UI can not be focused or operated by keyboard

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#15063]
### NuGet Product Used

Visual Studio Package Management UI

### Product Version

VS Main\12126.134 + NuGet Client Dev\7.11.0.24

### Worked before?

2.	It is not a regression since it is a new feature.

### Impact

It bothers me. A fix would be nice

#### Notes:
1.	The repro rate is 100%. 
#### Repro Steps:
1.	Create a C# Console App (.NET 10.0) in VS and click the 'Manage NuGet Packages...' from the context menu of the project in the Solution Explorer window.
2.	Go to the ‘Browse’ tab, select the source ‘nuget.org’ from the ‘Package source’ dropdown list and select a vulnerable package from the ‘Version’ dropdown list (for example, Newtonsoft.Json 12.0.2 in this instance).
3.	Install the package and keep pressing Tab key to see if the ‘Fix with GitHub Copilot’ link and ‘X’ button can be focused after the vulnerability InforBar is showing at the top of PM UI.
#### Expected Result:
Both the ‘Fix with GitHub Copilot’ link and ‘X’ button can be focused or operated by keyboard.
#### Actual Result:
Both the ‘Fix with GitHub Copilot’ link and ‘X’ button cannot be focused or operated by keyboard as the screenshot below.
<img width="1720" height="770" alt="Image" src="https://github.com/user-attachments/assets/c4c13713-c6a3-43a8-bf94-6ce80afd8215" />

### Verbose Logs

```shell

```

**Comment by nkolev92** [comment]
cc @donnie-msft

**Comment by donnie-msft** [comment]
Hi @v-luzh - I appreciate you calling out this behavior. In WPF, this requires special keystroke of `Shift + Alt`, which will immediately put focus inside the InfoBar within the tool window:

<img width="1120" height="128" alt="Image" src="https://github.com/user-attachments/assets/19def5ad-bb7d-4e46-a205-bdc9d169c736" />

For more information, see https://learn.microsoft.com/en-us/visualstudio/ide/reference/accessibility-tips-and-tricks?view=vs-2022#access-toolbars-by-using-keyboard-shortcuts

If you feel this is still an issue, please reopen. Thanks for finding!

**Comment by v-luzh** [comment]
Hi @donnie-msft, thanks for your info.
